pkg/assembler/nodes.go

```diff
@@ -41,6 +41,36 @@ func (an ArtifactNode) IdentifiablePropertyNames() []string {
 	return []string{"digest"}
 }
 
+// PackageNode is a node that represents an artifact
+type PackageNode struct {
+	Name   string
+	Digest string
```

Comment:
How do we ensure the digest is required?

Reply:
In some of the SPDX documents some of them don't have digests - and may not be defined for them (e.g. if they are just describing package versions). I think this will be an exercise once we ingest and then bring up in a laundry list to fix.

Reply:
we had more discussions around multiple and missing identifiers and i think we're going forward with using "purl" for now, since it works with the current dataset of syft SBOMs. This decision isn't quite set in stone and we want to discuss it further with the broader group and when @mihaimaruseac gets back. I've created an issue for now to revisit.